src/openforms/registrations/contrib/zgw_apis/plugin.py

```diff
@@ -71,6 +83,18 @@ class Meta:
             RoltypeOmschrijvingValidator(),
         ]
 
+    def validate(self, attrs):
```

Comment:
I was thinking that since `zgw_api_group` is not required, if it is not present, then `validate_zgw_api_group` will not run?

Reply:
Ah that could be, I'm not sure. Give it a try and see if it runs or not? If it doesn't, it'd make more sense to attach the error to this field instead of `non_field_errors` though :)